### ISSUE TITLE
Bintray and jcenter are shutting down on May 1

### ISSUE THREAD

**Issue by enricocid** [reddit/IndicatorFastScroll#45]
https://www.reddit.com/r/androiddev/comments/lbssfn/now_that_bintray_and_jcenter_are_shutting_down/

**Comment by DSteve595** [comment]
Yep, this is a little annoying. We'll move to Sonatype/Maven Central.

**Comment by bobekos** [comment]
any news to this?

**Comment by DSteve595** [comment]
Sorry, I haven't gotten around to it ☹️
JCenter changed their mind and is staying up. We won't do any future releases on JCenter though; next time a release is needed, it'll be on Maven Central.

**Comment by tonnyavery** [comment]
Any updates @DSteve595?
We are eager to switch our project off jcenter because there are issues once in a while accessing dependencies, like today

**Comment by DSteve595** [comment]
I've been slacking. Apologies again. Working on getting us a Sonatype account.

**Comment by tonnyavery** [comment]
I saw some other libs using jitpack, was a relatively fast setup.

**Comment by emmano** [comment]
Any updates on this? We are also trying to move off jCenter.

**Comment by tonnyavery** [comment]
@DSteve595 ?

**Comment by lamrak** [comment]
also waiting for migration... No updates here?

**Comment by msbouchedid** [comment]
I am also waiting for migration... JCenter is deprecated and I am trying to migrate our application from it. This is the last library in the project that is on JCenter. Any updates would be great or at least a timeline of when this could potentially get done.

Thank you in Advance!
Mario

**Comment by nicbell** [comment]
@msbouchedid I can suggest changing your JCenter references from

```gradle
jcenter()
```
To
```gradle
jcenter {
    content {
        includeGroup "com.reddit" // https://github.com/reddit/IndicatorFastScroll/issues/45
    }
}
```
This will mean when Gradle iterates through the repos and downloads libraries it will only look in JCenter for this library.
Should save build time and also prevent engineers from adding more JCenter dependencies.

**Comment by tonnyavery** [comment]
@DSteve595 Can we get some resolution to this ticket? What is the plan and when will the library be moved outside of jcenter?

**Comment by ethanwa** [comment]
Yes, we need an update to this quickly.

**Comment by odmfl** [comment]
I found this article, I don't know if it can help.  Can anyone also fork which will then merge to the main branch?

https://blog.shreyaspatil.dev/automate-library-publishing-to-maven-central-with-github-actions-workflow-dispatch

**Comment by raffaelgyr** [comment]
The solution provided by @nicbell used to work for a while but has also stopped working (at least for me). I guess at this point the only option is to replace this library.

Edit: Yes, I figured it out too by now. [JCenter has been completely sunset in August 2024](https://jfrog.com/blog/jcenter-sunset/) and that's why this stopped working.
